### ISSUE TITLE
virtualenv doesn't work if TCL is not installed

### ISSUE THREAD

**Issue by davidhyman** [pypa/virtualenv#945]
If (for any reason) the user did not elect to install TCL / TK during installation, we get the following error:

```
  File "c:\python35-32\lib\site-packages\virtualenv.py", line 329, in copyfileordir
    shutil.copy2(src, dest)
  File "c:\python35-32\lib\shutil.py", line 251, in copy2
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "c:\python35-32\lib\shutil.py", line 114, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\python35-32/tcl/tcl8.6'
```

The relevant code is that added to fix https://github.com/pypa/virtualenv/issues/93
https://github.com/pypa/virtualenv/pull/911

I propose the following patch (at ln1076) along the same lines as the code used elsewhere for the dll files (ln1277):

```
    for name in ['tcl', 'tk']:
        srcdir = src + '/tcl/' + name + libver
        dstdir = dest + '/tcl/' + name + libver
        if os.path.exists(srcdir):
            copyfileordir(srcdir, dstdir, symlink)
        else:
            logger.notify('omitted %s %s because it was not in the source Python distribution', name, libver)
```

This is a clear message and is forward-compatible should the user later elect to install tkinter and re-create the virtual environment.

**Comment by davidhyman** [comment]
Also at a guess this should originally have been written using os.path, rather than string concat?

**Comment by jayvdb** [comment]
Yes re using os.path , but `/` usually works on Windows as it is the `os.altsep` on Windows.

**Comment by Ivoz** [comment]
Hi there, 643140678aab52679d should have solved this issue, and is now in 15.0.3

**Comment by c0d3h4x0r** [comment]
> Hi there, [6431406](https://github.com/pypa/virtualenv/commit/643140678aab52679ded8f483003a3384c0e8253) should have solved this issue, and is now in 15.0.3

That did not fix it on Mac.

**Comment by gaborbernat** [comment]
Can you test with ``20.0.0b2``? I believe that should have fixed this.
